TS04-Plotting-Time-Series-R.Rmd

```diff
@@ -23,8 +23,7 @@ comments: false
 
 
 ##About
-This activity will walk you through the fundamentals of data manipulation and 
-basic plotting.
+This lesson will teach individuals how to conduct basic data manipulation and create plots of time series data using ggplot2.
```

Comment:
@mguarinello @lwasser   In the lessons we switch back and forth between "students", "participants", and "individuals" when describing the people doing these lessons .  Do we want to choose just one?